cli/fathom_web/commands/test.py

```diff
@@ -47,8 +47,10 @@ def model_from_json(weights, num_outputs, feature_names):
 @command()
 @argument('testing_file',
           type=File('r'))
+@argument('decision-threshold',
```

Comment:
What about calling it "confidence", since I think we call it that everywhere else. Or "score". But "decision" is new.

Reply:
That's a good point. I'll go with confidence.